update to NServiceBus 8.0.0-alpha.1896 (#5438)
Particular/docs.particular.net: nservicebus/hosting/override-machine-name_specifyingmachinename_core_[8,).partial.md

> To override the machine name of the endpoint, the endpoint has to be marked as uniquely idenfiable. Once that is done, the hostname for that instance can be specified.

Author: Borrowing "host machine name" from the upgrade guide change.

```suggestion
The host machine name of an endpoint instance may be overridden if the endpoint instance is uniquely identifiable:
```
